backend/main/strategies/staked_token_weighted_default.go

```diff
@@ -53,29 +53,30 @@ func (s *StakedTokenWeightedDefault) FetchBalanceFromSnapshot(
 	ftBalance.NewFTBalance()
 
 	if *strategy.Contract.Name == "FlowToken" {
-		if err := s.SC.GetAddressBalanceAtBlockHeight(
-			b.Addr,
-			b.BlockHeight,
-			ftBalance,
-			&strategy.Contract,
-		); err != nil {
-			log.Error().Err(err).Msg("Error fetching balance from snapshot client")
-			return err
-		}
+		// if err := s.DPS.GetAddressBalanceAtBlockHeight()
```

Comment:
Why are these lines commented out? Should they be deleted?

Reply:
oh whoops! i think i just forgot to finish implementing this strategy

Reply:
fixed!